syntax = "proto3";
message RuntimeData {
	enum State {
		STUDENT_CRASHED = 0;
		STUDENT_RUNNING = 1;
		STUDENT_STOPPED = 2;
		TELEOP = 3;

	}
	message SensorData {
		int32 type = 1;
		float value =2;
		int64 id = 3;
	}
	repeated SensorData sensor = 2;
	State robot_state = 1;
}
